configstack/stack.go

```diff
@@ -81,18 +81,23 @@ func (stack *Stack) Run(terragruntOptions *options.TerragruntOptions) error {
 func (stack *Stack) summarizePlanAllErrors(terragruntOptions *options.TerragruntOptions, errorStreams []bytes.Buffer) {
 	for i, errorStream := range errorStreams {
 		output := errorStream.String()
-		terragruntOptions.Logger.Infoln(output)
-		if strings.Contains(output, "Error running plan:") {
-			if strings.Contains(output, ": Resource 'data.terraform_remote_state.") {
-				var dependenciesMsg string
-				if len(stack.Modules[i].Dependencies) > 0 {
-					dependenciesMsg = fmt.Sprintf(" contains dependencies to %v and", stack.Modules[i].Config.Dependencies.Paths)
+
+		// We receive empty buffer if stack execution completed without errors
+		// But since we still want to capture full output if error occured - we check length of resulting string here
+		if len(output) > 0 {
```

Comment:
NIT: might be a tiny bit clearer to invert this. Something like:

```go
if len(output) == 0 {
  // If there were no errors, we'll get an empty buffer, so skip that to avoid logging lots of extra newlines
  continue
}

Reply:
@brikis98 yeah, your version looks cleaner 👍